protocols/gossipsub/src/peer_score.rs

```diff
@@ -208,6 +210,7 @@ impl PeerScore {
             params,
             peer_stats: HashMap::new(),
             peer_ips: HashMap::new(),
+            #[allow(deprecated)]
```

Comment:
You should be able to fix these similarly as described in https://github.com/libp2p/rust-libp2p/pull/3495#discussion_r1129112327.